os/arch/arm/src/artik053/src/artik053_tash.c

```diff
@@ -315,7 +319,11 @@ int board_app_initialize(void)
 			lldbg("ERROR: FAILED TO smart_initialize\n");
 			free(rambuf);
 		} else {
+#ifdef CONFIG_SMARTFS_MULTI_ROOT_DIRS
+			(void)mksmartfs(CONFIG_ARTIK053_RAMMTD_DEV_POINT, 1, false);
```

Comment:
1.  you can remove void type casting.
2. return value of mksmartfs should be checked.